packages/components/statistic/src/statistic.ts

```diff
@@ -0,0 +1,53 @@
+import { buildProps, definePropType } from '@element-plus/utils'
+import type { StyleValue } from 'vue'
+import type Statistic from './statistic.vue'
+export const statisticProps = buildProps({
+  decimalSeparator: {
+    type: String,
+    default: '.',
+  },
+  groupSeparator: {
+    type: String,
+    default: ',',
+  },
+  precision: {
+    type: Number,
+    default: null,
+  },
+  formatter: {
+    type: definePropType<(val: string | number) => string | number>(Function),
+  },
+  value: {
+    type: [String, Number],
+    default: null,
+  },
+  prefix: {
+    type: String,
+    default: '',
+  },
+  suffix: {
+    type: String,
+    default: null,
+  },
+  title: {
+    type: [String, Number],
+    default: '',
+  },
+  valueStyle: {
+    type: definePropType<StyleValue>([String, Object]),
+    default: () => ({}),
+  },
+  rate: {
```

Comment:
Is it better to use `groupStep` ?  /cc @kooriookami

Reply:
> Is it better to use `groupStep` ? /cc @kooriookami

Maybe `separatorCount` is more better.

Reply:
Different countries use different way to separate numbers. The `rate` prop can not satisfy all cases. We should abandon it.